amf-webapi/shared/src/main/scala/amf/plugins/document/WebApi.scala

```diff
@@ -9,7 +9,6 @@ import amf.plugins.domain.webapi.APIDomainPlugin
 
 import scala.scalajs.js.annotation.JSExportAll
 
-@JSExportAll
```

Comment:
useless export.

Reply:
This appears to be exported in old typings (line 1997), and is not present in the new typings. Is it due to this change?

Reply:
Yup. There is no way of referencing methods in an object if it is not exported with JSTopLevelExport

Reply:
Great